evm_arithmetization/tests/two_to_one_agg.rs

```diff
@@ -0,0 +1,237 @@
+        Node::Branch {
+            children,
+            value: vec![],
+        }
+        .into()
+    };
+
+    let receipt_0 = LegacyReceiptRlp {
+        status: true,
+        cum_gas_used: 21032.into(),
+        bloom: vec![0; 256].into(),
+        logs: vec![],
+    };
+    let mut receipts_trie = HashedPartialTrie::from(Node::Empty);
+    receipts_trie.insert(
+        Nibbles::from_str("0x80").unwrap(),
+        rlp::encode(&receipt_0).to_vec(),
+    )?;
+    let transactions_trie: HashedPartialTrie = Node::Leaf {
+        nibbles: Nibbles::from_str("0x80").unwrap(),
+        value: txn.to_vec(),
+    }
+    .into();
+
+    let trie_roots_after = TrieRoots {
+        state_root: expected_state_trie_after.hash(),
+        transactions_root: transactions_trie.hash(),
+        receipts_root: receipts_trie.hash(),
+    };
+    let inputs = GenerationInputs {
+        signed_txn: Some(txn.to_vec()),
+        withdrawals: vec![],
+        tries: tries_before,
+        trie_roots_after,
+        contract_code,
+        checkpoint_state_trie_root: HashedPartialTrie::from(Node::Empty).hash(),
+        block_metadata,
+        txn_number_before: 0.into(),
+        gas_used_before: 0.into(),
+        gas_used_after: 21032.into(),
+        block_hashes: BlockHashes {
+            prev_hashes: vec![H256::default(); 256],
+            cur_hash: H256::default(),
+        },
+    };
+
+    Ok(inputs)
+}
+
+fn dummy_inputs(inputs: &GenerationInputs) -> GenerationInputs {
+    GenerationInputs {
+        txn_number_before: inputs.txn_number_before + 1,
+        gas_used_before: inputs.gas_used_after,
+        gas_used_after: inputs.gas_used_after,
+        signed_txn: None,
+        withdrawals: vec![],
+        tries: TrieInputs {
+            state_trie: HashedPartialTrie::from(Node::Hash(inputs.trie_roots_after.state_root)),
+            transactions_trie: HashedPartialTrie::from(Node::Hash(
+                inputs.trie_roots_after.transactions_root,
+            )),
+            receipts_trie: HashedPartialTrie::from(Node::Hash(
+                inputs.trie_roots_after.receipts_root,
+            )),
+            storage_tries: vec![],
+        },
+        trie_roots_after: inputs.trie_roots_after,
+        checkpoint_state_trie_root: inputs.checkpoint_state_trie_root,
+        contract_code: Default::default(),
+        block_metadata: inputs.block_metadata.clone(),
+        block_hashes: inputs.block_hashes.clone(),
+    }
+}
+
+#[test]
+#[ignore]
+fn test_two_to_one_aggregation() -> anyhow::Result<()> {
+    let all_stark = AllStark::<F, D>::default();
+    let config = StarkConfig::standard_fast_config();
+
+    let inputs0 = simple_transfer(1)?;
+    let dummy0 = dummy_inputs(&inputs0);
+    let inputs1 = simple_transfer(2)?;
+    let dummy1 = dummy_inputs(&inputs1);
+
+    // Preprocess all circuits.
+    let all_circuits = AllRecursiveCircuits::<F, C, D>::new(
+        &all_stark,
+        &[8..17, 8..15, 8..18, 8..15, 8..10, 8..13, 8..20],
```

Comment:
nit: let's at least use the start ranges of the `empty_txn_list` example so that it doesn't require a huge amount of RAM when trying it locally :) 
```suggestion
        &[16..17, 9..15, 12..18, 14..15, 9..10, 12..13, 17..20]
```